src/Observables.jl

```diff
@@ -252,12 +250,8 @@ function Base.setindex!(observable::Observable, channel::Channel; notify=x->true
     end
 end
 
-function Base.setindex!(observable::AbstractObservable, val; notify=x->true)
-    Base.setindex!(observe(observable), val; notify=notify)
-end
-
-function setexcludinghandlers(observable::AbstractObservable, val, pred=x->true)
```

Comment:
I know that this is internal, but I guess we could keep something along the lines of

```julia
setexcludinghandlers(obs::AbstractObservable, val) = observe(obs).val = val
```

so that the "silent update" is not done by touching an internal field as in https://github.com/JuliaGizmos/Observables.jl/pull/56#issuecomment-754059842. In particular, the `observe(obs)` pattern is needed because Interact widgets are `AbstractObservable`s, whose "concrete observable" can be accessed with `observe(obs)`. As mentioned in that comment, `setindex!` would then just by `setexcludinghandlers` followed by `notify!`.

Reply:
Added.